azure-pipelines.yml

```diff
@@ -78,3 +78,40 @@ jobs:
         ArtifactName: 'Windows Desktop Spanish'
       continueOnError: true
       condition: not(succeeded())
+
+- job: Ubuntu 16.04
+  strategy:
+    maxParallel: 4
+    matrix:
+      Debug:
+        _configuration: Debug
+        _codeCoverage: True
+      Release:
+        _configuration: Release
+        _codeCoverage: False
+  pool:
+    name: NetCorePublic-Pool
+    queue: 'BuildPool.Ubuntu.1604.amd64.Open'
```

Comment:
I took this from dotnet/roslyn. Are there other pools available? Ubuntu 16.04 is quite old; I would prefer to use something more recent here (maybe as an alternative?).

Reply:
@JoeRobich @dotnet/roslyn-infrastructure for guidance.

Reply:
That is the most recent ubuntu arcade queue for now.

Reply:
@JoeRobich What would be a good way to test this? Seems the tests are not running for this PR.

Reply:
A space in the job name is stopping the pipeline from running.

> Job Ubuntu 16.04 has an invalid name. Valid names may only contain alphanumeric characters and '_' and may not start with a number.